### ISSUE TITLE
assertj configuration is generated too late

### ISSUE THREAD

**Issue by SerialVelocity** [opengl-8080/assertjGen-gradle-plugin#4]
The `assertj` configuration is generated inside a `doFirst` block which is after some plugins try to read the configuration.

It also stops people extending the `assertj` configuration.

**Comment by SerialVelocity** [comment]
This affected us because a multi-project repo cannot depend on these files as they aren't exposed by the time the intelliJ plugin (or maybe gradle too) tries to depend on them.

**Comment by opengl-8080** [comment]
I'm sorry for the late reply.
Thank you for your report.

I made a draft jar file. (Please unzip `assertjGen-gradle-plugin-1.1.3_draft.zip`)
[assertjGen-gradle-plugin-1.1.3_draft.zip](https://github.com/opengl-8080/assertjGen-gradle-plugin/files/790265/assertjGen-gradle-plugin-1.1.3_draft.zip)

Is this what you want?

### How to use
```groovy
buildscript {
    dependencies {
        classpath files("path/to/assertjGen-gradle-plugin-1.1.3_draft.jar")
    }
}

apply plugin: "com.github.opengl-BOBO.assertjGen"
```

**Comment by SerialVelocity** [comment]
Sorry for my late response, but we used manually generated files in the end since our objects weren't in the "bean" style that the generator expects. I'll see whether I can create a test project soon but that may only be in a few weeks.

**Comment by opengl-8080** [comment]
Ok. I wait for you.

**Comment by opengl-8080** [comment]
This bug is fixed to resolve [#7 problem](https://github.com/opengl-8080/assertjGen-gradle-plugin/issues/7).
